File: JDApp-Java/other/src/main/java/com/jd/other/viewholder/OtherViewHolder.java
package com.jd.other.viewholder;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import com.jd.core.base.adapter.BaseListViewAdapter;
import com.jd.other.R;

import java.util.Map;

public class OtherViewHolder implements BaseListViewAdapter.BaseViewHolder {
    ImageView imageView;
    TextView textView1;
    TextView textView2;

    @Override
    public int layout_id() {
        return R.layout.list_item;
    }

    @Override
    public void onCreateView(View view) {
        imageView = (ImageView) view.findViewById(R.id.image1);
        textView1 = (TextView) view.findViewById(R.id.text1);
        textView2 = (TextView) view.findViewById(R.id.text2);
    }

    @Override
    public void onStart(Object data) {
        Map<String,Object> item = (Map<String,Object>)data;
//        imageView.setImageResource(Datas.get(i).getImageId());
        textView1.setText(item.get("title").toString());
    }
}

File: JDApp-Kotlin/other/src/main/java/com/jd/other/viewholder/OtherViewHolder.kt
package com.jd.other.viewholder

import android.view.View
import android.widget.ImageView
import android.widget.TextView

import com.jd.core.base.adapter.BaseListViewAdapter
import com.jd.other.R

class OtherViewHolder : BaseListViewAdapter.BaseViewHolder {
    lateinit internal var imageView: ImageView
    lateinit internal var textView1: TextView
    lateinit internal var textView2: TextView

    override fun layout_id(): Int {
        return R.layout.list_item
    }

    override fun onCreateView(view: View?) {
        imageView = view!!.findViewById<View>(R.id.image1) as ImageView
        textView1 = view.findViewById<View>(R.id.text1) as TextView
        textView2 = view.findViewById<View>(R.id.text2) as TextView
    }

    override fun onStart(data: Any?) {
        val item = data as Map<String, Any>?
        //        imageView.setImageResource(Datas.get(i).getImageId());
        textView1.text = item!!["title"]!!.toString()
    }
}
